### PR TITLE
Statistics (EXPOSUREAPP-3674)

### PR DESCRIPTION
**Pull request by d4rken**
Don't squash this.

### CODE REVIEW

**Review by harambasicluka — DISMISSED** [review]

**Comment by harambasicluka** [comment]
Please do not merge until we have the final decision.

**Review by harambasicluka — DISMISSED** [review]

**Review by ralfgehrer — APPROVED** [review]

**Review by harambasicluka — APPROVED** [review]

**Pull request merged**
